lib/spec_file_generator.js

```diff
@@ -19,10 +19,17 @@ function sorted(items) {
 }
 
 function dependenciesToSources(deps) {
+  deps.sort((a, b) => {
+    if (a[0] === b[0]) {
+      return a[1].localeCompare(b[1]);
+    } else {
+      return a[0].localeCompare(b[0]);
+    }
+  });
   const sources = deps.map((dependency, index) => {
     return `Source${index}: ${npmUrl(dependency[0], dependency[1])}`;
```

Comment:
When using `dependenciesToProvides` this should become the more readable:

```suggestion
    return `Source${index}: ${npmUrl(dependency['name'], dependency['version'])}`;
```